docs/repositories-configure/codacy-configuration-file.md

```diff
@@ -27,25 +27,31 @@ To use a Codacy configuration file:
     ---
     engines:
       rubocop:
+        enabled: true
         exclude_paths:
-          - config/engines.yml
-        base_sub_dir: test/baseDir
+          - "config/test.yml"
+        base_sub_dir: "test/baseDir"
       duplication:
+        enabled: true
         exclude_paths:
-          - config/engines.yml
-      metric:
+          - "config/test.yml"
+        config:
+          languages:
+            - "ruby"
+      metrics:
+        enabled: true
         exclude_paths:
-          - config/engines.yml
+          - "config/test.yml"
```

Comment:
We had a mistake here: before it was `metric:` instead of `metrics:`.

Do you confirm that this is the right syntax @lolgab @franciscodua @josemiguelmelo? I figured this out by looking at test files in the codacy/codacy-analysis-cli repository, for example:

https://github.com/codacy/codacy-analysis-cli/blob/master/core/src/test/resources/com/codacy/analysis/core/configuration/codacy.yaml.

Reply:
You're right. It's `metrics`
https://github.com/codacy/codacy-analysis-cli/blob/master/core/src/main/scala/com/codacy/analysis/core/tools/MetricsTool.scala#L21